src/Delete.js

```diff
@@ -0,0 +1,19 @@
+import './App.css'
+
+function Delete() {
```

Comment:
Delete is pretty generic & can be confused with `delete`. Try not to use very generic names

Reply:
changed to DeleteEvent